sections/semantics-interactive-elements.include

```diff
@@ -1,5 +1,23 @@
 <section>
 
+<!--
+
+    Interactive elements
+
+  This source produces section 4.11: Interactive elements
+  http://w3c.github.io/html/interactive-elements.html
+
+  It covers
+
+  - media elements:
```

Comment:
I don't think so, I think it covers details/summary, dialog, and commands

Reply:
wow. sorry about that.  I don't know what happened there. 

Just pushed a fix!